### PR TITLE
Date transform and found country in arrow and string back complite.

### PR DESCRIPTION
**Pull request by oiskritsky**
Date transform and found country in arrow and string back complite.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -170,18 +170,48 @@
...
 
 const date = "2020-08-10";
 const date1 = "2021-12-26";
 
 const datetransform = (str) => {
   let arr = str.split("-");
   arr.reverse();
+
   if (arr[0] > 0 && arr[0] <= 31 && arr[1] >= 1 && arr[1] <= 12) {
     return (result = arr.join("."));
```
Comment (ghost) [inline js/main.js]: `result` doesn't define.

File: js/main.js [hunk 2]
```diff
@@ -170,18 +170,48 @@
...
+  { country: "Marocco", city: "Ourika", hotel: "Rokoko Hotel" },
+  { country: "Germany", city: "Berlin", hotel: "Hotel Rehberge Berlin Mitte" },
+];
+
+let search = prompt("Какую страну вы ищите?", "Germany");
+
+const getCountryData = (x, arr) => {
+  let result = "";
+  arr.forEach((element) => {
+    if (element.country == x) {
```
Comment (ghost) [inline js/main.js]: What if I true to find by city or hotel ?

File: js/main.js [hunk 3]
```diff
@@ -170,18 +170,48 @@
...
+      return (result += `Страна: ${element.country}, город: ${element.city}, отель: ${element.hotel} `);
+    }
+  });
+  return result;
```
Comment (ghost) [inline js/main.js:217]: Result should be an array.

File: js/main.js [hunk 4]
```diff
@@ -170,18 +170,48 @@
...
+  { country: "Germany", city: "Berlin", hotel: "Hostel Friendship" },
+  { country: "Indonesia", city: "Bali", hotel: "Ubud Bali Resort&SPA" },
+  { country: "Netherlands", city: "Rotterdam", hotel: "King Kong Hostel" },
+  { country: "Marocco", city: "Ourika", hotel: "Rokoko Hotel" },
+  { country: "Germany", city: "Berlin", hotel: "Hotel Rehberge Berlin Mitte" },
+];
+
+let search = prompt("Какую страну вы ищите?", "Germany");
+
+const getCountryData = (x, arr) => {
```
Comment (ghost) [inline js/main.js]: Check description for the task.

File: js/main.js (hunk 3 above)
Comment (oiskritsky) [reply js/main.js:217]: done

File: js/main.js (hunk 4 above)
Comment (oiskritsky) [reply js/main.js]: done

File: js/main.js (hunk 1 above)
Comment (oiskritsky) [reply js/main.js]: done

**Pull request merged**